src/raytracer/getTgayReflectionLoss.m

```diff
@@ -0,0 +1,107 @@
+% WARRANTY OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE,
+% NON-INFRINGEMENT AND DATA ACCURACY. NIST NEITHER REPRESENTS NOR WARRANTS
+% THAT THE OPERATION OF THE SOFTWARE WILL BE UNINTERRUPTED OR ERROR-FREE,
+% OR THAT ANY DEFECTS WILL BE CORRECTED. NIST DOES NOT WARRANT OR MAKE ANY
+% REPRESENTATIONS REGARDING THE USE OF THE SOFTWARE OR THE RESULTS THEREOF,
+% INCLUDING BUT NOT LIMITED TO THE CORRECTNESS, ACCURACY, RELIABILITY,
+% OR USEFULNESS OF THE SOFTWARE.
+%
+% You are solely responsible for determining the appropriateness of using
+% and distributing the software and you assume all risks associated with
+% its use,including but not limited to the risks and costs of program
+% errors, compliance with applicable laws, damage to or loss of data,
+% programs or equipment, and the unavailability or interruption of
+% operation. This software is not intended to be used in any situation
+% where a failure could cause risk of injury or damage to property.
+% The software developed by NIST employees is not subject to copyright
+% protection within the United States.
+%
+% 2020-2021 NIST/CTL (neeraj.varshney@nist.gov)
+
+% Get reflectances for Vertical and Horizontal polarization
+reflectionCoefficient = getReflectance(MaterialLibrary,materialIndex,multipath);
+% Calculate reflection loss. It is the sum (in dB) of the mean
+% of the Vertical and Horizontal reflectances for each reflection.
+reflectionLoss = prod(mean(reflectionCoefficient));
+reflectionLossdB = -20*log10(abs(reflectionLoss)); 
+end
+
+function reflectionCoefficient = getReflectance(MaterialLibrary,...
+                                                materialIndex,multipath)
+% This function calculates reflection coefficients
+incidentAngle = angleOfIncident(multipath);
+orderReflection = multipath(1,1);
+reflectionCoefficient = ones(2, orderReflection);
+for reflectionOrderIndex = 1:orderReflection
+    if reflectionOrderIndex == 1
+        reflectionMaterialIndex = materialIndex(1,1);
+    elseif  reflectionOrderIndex == 2
+        reflectionMaterialIndex = materialIndex(1,2);
+    else
+        error(strcat('Incident angles are obtained till second order ',...
+        'reflection. Thus, order of reflection cannot be considered ',... 
+        'higher than second order reflection.'));
+    end
+    % Use Fresnel equation to derive power reflectivity                
+    relativePermittivity = MaterialLibrary.RelativePermittivity...
+                            (reflectionMaterialIndex); 
+    aor = incidentAngle(reflectionOrderIndex);
+    B_h =  relativePermittivity - sind(aor)^2;                
+    B_v = (relativePermittivity - sind(aor)^2)/relativePermittivity^2; 
+    reflectionCoefficient(:, reflectionOrderIndex) = [ ...
+        (cosd(aor) - sqrt(B_v))/(cosd(aor) + sqrt(B_v)); ... % Vertical
+        (cosd(aor) - sqrt(B_h))/(cosd(aor) + sqrt(B_h))];    % Horizontal
+end     
+end
+
+function incidentAngle = angleOfIncident(multipath) 
```

Comment:
*angleOfIncidence
missing doc

Reply:
not fixed?

Reply:
you mean output variable?

Reply:
okay! you are about function name! wait let me correct

Reply:
done